### ISSUE TITLE
private ground notification needs improvement

### ISSUE THREAD

**Issue by schnedann** [osmandapp/OsmAnd#20544]
### Describe the idea (required)

One of my favorites - work - ends in a non public car park. Sop every morning OSMAND asks if I REALLY want to navigate to private ground. And the answer is ALWAYS yes.

The Idea now is to only ask once - when the target is selected the first time - and when the answer is yes just never ask again.

### Tell us about the expected behaviour (required)

When a target on private ground is selected, ask if one wants to navigate to it.

If a target then is stored as favourite or is in the list of previous targets, and the answer was yes in first place, then just do not ask again. Simply assume one is conscious about the consent to navigate to private ground.

### Tell us about alternatives you've considered (required)

one could mark private ground target with a symbol or color, so at re-selection there is a hint but no dialog

**Comment by Rowin63** [comment]
Perhaps something like the opposite of the option (mark) "avoid road"?

**Comment by scaidermern** [comment]
You can also move your favorite to the entrance so that it is located right in front of the private road but not inside it.

**Comment by schnedann** [comment]
well, thats maybe is a workaround, but a software that would just behave more sane is preferable

**Comment by sonora** [comment]
So you need navigation to find to work, _every morning_ ... ? :wink::wink::wink:

**Comment by schnedann** [comment]
In fact I use OSMAND / navigation for every drive, even short ones.

* Speed indication
* fast overview of alternatives when there is a traffic problem
* hints about when I need to change to another street --> so I can concentrate on other things more
* I have a feeling due the estimated time, the real time and on how the estimated time is adjusting...

**Comment by vshcherb** [comment]
Yes I think it make sense to remember it however it would be difficult to reset. It could be linked to Favorite setting

**Comment by schnedann** [comment]
I have no knowledge on OSMAND Code, but I would assume a target shown in the list of previous targets or the favourites is a kind of structure type? with name, Position coordinates, POI-class,... so adding another member to the struct and set it should not be a problem. if then the target is deleted from the favourites or the list of previous targets, its "reseted" automatically. And if there is a good design that structure should be the same for favourites and the list of previous targets...

Does that sound correct?
